index.d.ts

```diff
@@ -22,7 +22,8 @@ interface Props {
   allowDragging?: boolean
   showBackdrop?: boolean
   backdropOpacity?: number
-  friction?: number
+  friction?: number,
+  contentStyle?: any,
```

Comment:
Should be:
```suggestion
  contentStyle?: StyleProp<ViewStyle>
```
And don't forget to `import { StyleProp, ViewStyle } from react-native`.